packages/workbox-build/src/lib/utils/filter-files.js

```diff
@@ -1,14 +1,55 @@
+const constants = require('../constants');
+const logHelper = require('../log-helper');
+const modifyUrlPrefixTranform = require('./modify-url-prefix-transform');
+const noRevisionForUrlsMatchingTransform =
+  require('./no-revision-for-urls-matching-transform');
 const path = require('path');
 
-const logHelper = require('../log-helper');
-const constants = require('../constants');
-const modifyUrlPrefixes = require('./modify-url-prefix');
+/**
+ * A `ManifestTransform` function can be used to modify the modify the `url` or
+ * `revision` properties of some or all of the
+ * {@link module:workbox-build#ManifestEntry|ManifestEntries} in the manifest.
+ *
+ * Setting the `revision` property of an entry to a "falsey" value will cause
```

Comment:
Why not just return an object without a revision key, otherwise it must be a string with at least one character?


I always get nervous with "falsey" values because it just introduces possibility of a simple mistake in user code that just results in unexpected behavior.